<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>551</width>
     <height>286</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="5">
     <widget class="QCheckBox" name="checkBox_displayNewline">
      <property name="text">
       <string>自动换行</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QComboBox" name="comBox_uartDps">
      <item>
       <property name="text">
        <string>115200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9600</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="pushButton_uartDisconnect">
      <property name="text">
       <string>关闭串口</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="MyComboBox" name="comBox_uartPort"/>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>串口</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>波特率</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QRadioButton" name="radioButton_displayHEX">
      <property name="text">
       <string>HEX显示</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QRadioButton" name="radioButton_displayASCII">
      <property name="text">
       <string>ASCII显示</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="5">
     <widget class="QPushButton" name="pushButton_clear_TBinput">
      <property name="text">
       <string>清空接收</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="pushButton_uartConnect">
      <property name="text">
       <string>打开串口</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="6">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0" colspan="7">
     <widget class="QTextBrowser" name="textBrowser_intput"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>551</width>
     <height>121</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="4">
     <widget class="QPushButton" name="LEFT">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>左摇杆</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QPushButton" name="RIGHT">
      <property name="text">
       <string>右摇杆</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="4">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="KEY1">
        <property name="text">
         <string>KEY1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="KEY2">
        <property name="text">
         <string>KEY2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="KEY3">
        <property name="text">
         <string>KEY3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="KEY4">
        <property name="text">
         <string>KEY4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="KEY5">
        <property name="text">
         <string>KEY5</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="2">
     <widget class="QPlainTextEdit" name="textEdit_output">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>5</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>直接发送</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QRadioButton" name="radioButton_sendoutASCII">
      <property name="text">
       <string>ASCII发送</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QRadioButton" name="radioButton_sendoutHEX">
      <property name="text">
       <string>HEX发送</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="5">
     <widget class="QPushButton" name="buttom_sendout">
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>551</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="1" column="0" rowspan="2">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButton_23_1">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="radioButton_23_2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="radioButton_23_3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButton_23_4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="radioButton_23_5">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QRadioButton" name="radioButton_23_6">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="2" rowspan="2" colspan="2">
     <widget class="QPushButton" name="pushButton_inject">
      <property name="text">
       <string>给药</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>3×4孔板</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4" rowspan="2" colspan="2">
     <widget class="QPushButton" name="pushButton_absorb">
      <property name="text">
       <string>采样</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>50</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="4">
     <widget class="QPlainTextEdit" name="plainTextEdit_volume">
      <property name="plainText">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>2×3孔板</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" rowspan="2">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="3">
       <widget class="QRadioButton" name="radioButton_34_8">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QRadioButton" name="radioButton_34_11">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="radioButton_34_3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="radioButton_34_2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="radioButton_34_10">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QRadioButton" name="radioButton_34_4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="radioButton_34_6">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButton_34_5">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButton_34_1">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QRadioButton" name="radioButton_34_7">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="radioButton_34_9">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QRadioButton" name="radioButton_34_12">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>μL</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>体积（最小1μL）</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyComboBox</class>
   <extends>QComboBox</extends>
   <header>mycombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
